modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1827,6 +1900,7 @@ private final double getViewportBreadth() {
     private double viewportLength;
     void setViewportLength(double value) {
         this.viewportLength = value;
+        this.absoluteOffset = getPosition() * (estimatedSize -viewportLength);
```

Comment:
Minor: spacing

Reply:
solved this and the previous spacing comments.